README.md

```diff
@@ -21,247 +18,160 @@ techniques for pre-rendering cacheable pages.
+that you can use for development and testing.
 
-### If you want to run the Firestore Emulator locally
-
-Choose the port you want to use for the [Firestore Emulator], for example `9999`.
-
-> Note: If you choose a port that is not `9999` for this demo, you will need to update the `host` property in `database.ts` to match your selected port.
-
-Export the `FIRESTORE_EMULATOR_HOST` environment variable:
-
-```bash
-export FIRESTORE_EMULATOR_HOST="localhost:9999"
-```
-
-Start the emulator:
-
-```bash
-gcloud emulators firestore start --host-port="$FIRESTORE_EMULATOR_HOST" --project=demo-test
-```
-
-> When the exported `FIRESTORE_EMULATOR_HOST` environment variable is set, the
-> Firestore client will automatically use it to connect to the emulator.
-
-Press `Ctrl-C` when you want to stop the Firestore emulator.
-
-> For more detail, see the docs for [Emulate Firestore locally](https://cloud.google.com/firestore/docs/emulator).
+> The Google Cloud Firestore emulator requires a Java 8+ JRE installed and on
+> your system PATH
+>
+> If you need to install Java, you can get it from here:
+> https://adoptium.net/
+>
+> - To choose a specific version, to to: https://adoptium.net/temurin/releases/
+> - If you're on a Mac with an Apple Silicon M1 or M2 chip, choose the `aarch64` architecture.
+> - Choose `JRE` instead of the full `JDK` package unless you plan to do Java
+>   development.
 
-## Tests
+### Run the app locally
 
-```bash
-npm test
-npm run test:watch
-```
+1. Start the Firestore Emulator
 
-[ts-jest](https://kulshekhar.github.io/ts-jest/) is installed as a dev
-dependency to support writing tests in TypeScript. The jest configuration
-(`jest.config.js`) sets `preset: 'ts-jest'` to enable this.
+   Choose the port you want to use for the [Firestore Emulator], for example `9999`.
 
-Jest will run tests located in `__test__` directories (for example:
-`src/lib/__test__/database.test.ts`).
+   > Note: If you choose a port that is not `9999` for this demo, you will need to update the `host` property in `database.ts` to match your selected port.
 
-## Deployment
+   Export the `FIRESTORE_EMULATOR_HOST` environment variable:
 
-Ready to deploy? Check out how to do that [here](/terraform/README.md). 
+   ```bash
+   export FIRESTORE_EMULATOR_HOST="localhost:9999"
+   ```
 
-## Prerequisites
+   When the exported `FIRESTORE_EMULATOR_HOST` environment variable is set, the
+   Firestore client will automatically use it to connect to the emulator.
 
-You need a Google Cloud account if you want to open the demo in [Cloud Shell]
-(a free online development and operations environment with an integrated editor)
-and deploy it.
+   To start the emulator:
 
-If you want to explore the developer experience **using your own machine**, you
-will need to install the Google Cloud CLI.
+   ```bash
+   gcloud emulators firestore start --host-port="$FIRESTORE_EMULATOR_HOST" --project=demo-test
```

Comment:
"demo-test" isn't a project anyone would have. I found I didn't need this flag while trying this README. Suggest removing it. 
```suggestion
   gcloud emulators firestore start --host-port="$FIRESTORE_EMULATOR_HOST"
```